### ISSUE TITLE
Scene Text Recognition

### ISSUE THREAD

**Issue by ptpuyen1511** [ptpuyen1511/my_notes#8]
## TL;DR
Some notes from VINAI's seminar: Scene Text Recognition

### Dataset
- SynthText:
  * 80k synthetic images from 8k background images
- ICDAR15
  * 720x1280 images captured by Gooogle Glasses
  * 1k train, 0.5k tests images
- ICDAR17
   * 9 languages, 6 scripts
   * 7.2k train, 1.8k validation, 9k test images
- Coco-Text
   * 63k images with 173k text annotations
- Total-Text
   * Text of various shapes: horizontal, multi-oriented, curved
   * 1255 train, and 300 test images
- CTW1500
   * curved text
   * 1k train, 0.5k test
- Chinese Street View
   * 30k fully annotated
   * 400k weakly annotated
- MSRA-T500
   * 300 train, 200 test English and Chinese
   * Text instances are labeled at text-line level

#### Real-time Scene Text Detection with Differentiable Binarization (AAAI 2020)
![image](https://user-images.githubusercontent.com/18265363/79473789-ce3ace00-802f-11ea-8c14-7cb6a2075fdf.png)
![image](https://user-images.githubusercontent.com/18265363/79473848-e1e63480-802f-11ea-998b-e88824517b7e.png)
![image](https://user-images.githubusercontent.com/18265363/79474055-2671d000-8030-11ea-9cff-85133e5b0efc.png)
![image](https://user-images.githubusercontent.com/18265363/79474295-69cc3e80-8030-11ea-9eb5-b344b4e8931d.png)
![image](https://user-images.githubusercontent.com/18265363/79474358-7e103b80-8030-11ea-9d33-a2ee34976a72.png)
![image](https://user-images.githubusercontent.com/18265363/79474624-d8110100-8030-11ea-8592-e39622687b3c.png)
![image](https://user-images.githubusercontent.com/18265363/79474800-10b0da80-8031-11ea-80b8-5aee19682b25.png)

#### Convolutional Character Networks, ICCV'19
![image](https://user-images.githubusercontent.com/18265363/79476880-be24ed80-8033-11ea-8268-837d4a4cde3b.png)
![image](https://user-images.githubusercontent.com/18265363/79476980-d5fc7180-8033-11ea-92bb-625e195bb42a.png)
![image](https://user-images.githubusercontent.com/18265363/79477234-2d9add00-8034-11ea-81fd-b886a545b5fd.png)
![image](https://user-images.githubusercontent.com/18265363/79477390-5a4ef480-8034-11ea-9471-3b2b22f5858e.png)
![image](https://user-images.githubusercontent.com/18265363/79477425-633fc600-8034-11ea-9d89-e2e42b3dfb02.png)
![image](https://user-images.githubusercontent.com/18265363/79477490-7783c300-8034-11ea-8ead-1184a6a879f2.png)
![image](https://user-images.githubusercontent.com/18265363/79477737-c3cf0300-8034-11ea-8b00-6655e7ee240c.png)

Colab: https://www.dropbox.com/s/mxvoku6rh8bzq06/CharNet.zip?dl=0

#### ABCNet: Real-time Scene Text Sptting with adaptive Bezier-Curve Network
![image](https://user-images.githubusercontent.com/18265363/79480267-1958df00-8038-11ea-9eea-e0bdfcbc7113.png)
![image](https://user-images.githubusercontent.com/18265363/79480364-2ece0900-8038-11ea-8ac9-3c703128f72c.png)
![image](https://user-images.githubusercontent.com/18265363/79480431-40afac00-8038-11ea-9a9b-85c21d1aa308.png)
![image](https://user-images.githubusercontent.com/18265363/79480674-9d12cb80-8038-11ea-9e33-402558e89d2c.png)
![image](https://user-images.githubusercontent.com/18265363/79480925-fe3a9f00-8038-11ea-9b10-ad2a07e17bde.png)
![image](https://user-images.githubusercontent.com/18265363/79481013-20342180-8039-11ea-9ebe-813d3b0c87d0.png)
![image](https://user-images.githubusercontent.com/18265363/79481158-4d80cf80-8039-11ea-95e3-b6f52ffbbf6b.png)
![image](https://user-images.githubusercontent.com/18265363/79481315-828d2200-8039-11ea-825c-d63b3746a6a8.png)
![image](https://user-images.githubusercontent.com/18265363/79481558-def04180-8039-11ea-956e-7b139b65e669.png)
![image](https://user-images.githubusercontent.com/18265363/79481611-f2031180-8039-11ea-9246-d89ef4fb0ba0.png)
![image](https://user-images.githubusercontent.com/18265363/79481891-473f2300-803a-11ea-9a86-33a4dae08fe4.png)
